src/react/components/common/keyboardManager/keyboardRegistrationManager.ts

```diff
@@ -4,7 +4,18 @@ import Guard from "../../../../common/guard";
  * A map of keyboard event registrations
  */
 export interface IKeyboardRegistrations {
-    [key: string]: KeyboardEventHandler[];
+    [keyEventType: string]: {
+        [key: string]: KeyboardEventHandler[],
+    };
+}
+
+/**
+ * Types of Key events supported by registration manager
+ */
+export enum KeyEventType {
+    KeyDown = "keydown",
```

Comment:
Nit: camelCase for value?

Reply:
I think the event needs to be all lowercase. I was just following the convention that was already here. @wbreza any thoughts?